share/spice/mtg/mtg.js

```diff
@@ -0,0 +1,61 @@
+(function (env) {
+    "use strict";
+    env.ddg_spice_mtg = function(api_result){
+
+
+        if (!api_result || api_result.error || api_result.length == 0) {
+            return Spice.failed('mtg');
+        }
+
+
+        var infoboxItems = {
+            manaCost: "Mana Cost",
+            convertedManaCost: "Converted Mana Cost",
+            cardSetName: "Card Set Name",
+            type: "Type",
+            subType: "Sub Type",
+            power: "Power",
+            toughness: "Toughness",
+            loyalty: "Loyalty",
+            rarity: "Rarity",
+            artist: "Artist",
+            cardSetId: "Card Set Id",
+            token: "Token",
+            promo: "Promo"
+        };
+
+        var infoboxData = [{
+            heading: 'Card Details',
+        }];
+
+        $.each(infoboxItems, function(key, value){
+            if (api_result[0][key]){
+                infoboxData.push({
+                    url: "http://www.mtgdb.info/cards/" + api_result[0].id,
```

Comment:
Should be in the normalize function instead of in here. Shouldn't have a url here.